src/runtime/graph/graph_runtime.cc

```diff
@@ -87,6 +87,22 @@ class GraphRuntime : public ModuleNode {
     LOG(FATAL) << "cannot find " << name << " among input";
     return -1;
   }
+#ifdef TVM_GRAPH_RUNTIME_DEBUG
+  /*!
+   * \brief Get the node index given the name of node.
+   * \param name The name of the node.
+   * \return The index of node.
+   */
+  int GetNodeIndex(const std::string& name) {
+    for (uint32_t nid = 0; nid< nodes_.size(); ++nid) {
+      if (nodes_[nid].name == name) {
+        return static_cast<int>(nid);
+      }
+    }
+    LOG(FATAL) << "cannot find " << name << " among nodex";
```

Comment:
merge this macro region to DebugGetNodeOutput

Reply:
Done